draft-ietf-core-problem-details.md

```diff
@@ -13,224 +17,408 @@ stand_alone: yes
+This specification defines a generic problem type container with only a
+minimal set of attributes to makes it usable.
 
-Note that CoRAL documents are exchanged in CoRAL binary format (Section 3 of {{!I-D.ietf-core-coral}}) in practice. This includes the use of {{?I-D.ietf-core-href}} as an alternative to URIs that is optimized for constrained nodes.
+It is expected that applications will extend the base format by defining new
+attributes.
 
-The example in {{fig-example-minimalist}} has the most compact representation.  It avoids any non-mandatory element.  This is suitable for a constrained receiver that happens to have precise knowledge of the semantics associated with the "type".
+These new attributes fall into two categories: generic and application
+specific.
 
-~~~
-#using pd = <http://example.org/vocabulary/problem-details#>
-#using ex = <http://vocabulary.private-api.example/#>
+Generic attributes will be allocated in the `standard-problem-detail-entries`
+slot according to the registration procedure defined in {{new-spdk}}.
 
-pd:type         ex:unknown-key-id
-~~~
-{: #fig-example-minimalist title="Minimalist"}
+Application-specific attributes will be allocated in the
+`custom-problem-detail-entries` slot according to the procedure described in
+{{new-cpdk}}.
 
-The example in {{fig-example-full-fledged}} has all the mandatory as well as the optional elements populated.  This format is appropriate for a less constrained receiver (for example, an edge gateway forwarding to a log server that needs to gather as much contextual information as possible, including the problem "headline", details about the error condition, and an error-specific instance URL).
+## Standard Problem Detail Entries {#new-spdk}
 
-~~~
-#using pd = <http://example.org/vocabulary/problem-details#>
-#using ex = <http://vocabulary.private-api.example/#>
+Beyond the Standard Problem Detail keys defined in {{cddl}}, additional
+Standard Problem Detail keys can be registered in the
+`standard-problem-detail-entries` slot (see {{iana-spdk}}).
 
-pd:type         ex:unknown-key-id
-pd:title        "unknown key id"
-pd:detail       "Key with id 0x01020304 not registered"
-pd:instance     <https://private-api.example/errors/5>
+Standard Problem Detail keys are negative integers, so they never can
+conflict with Custom Problem Detail keys defined for a problem type
+(which are unsigned integers or URIs.)
+
+In summary, the keys for Standard Problem Detail entries are in a
+global namespace that is not specific to a particular application domain.
+
+Consumers of a Concise Problem Details instance MUST ignore any Standard Problem
+Detail entries that they do not recognize; this allows problem types to evolve.
+
+## Custom Problem Detail Entries {#new-cpdk}
+
+Applications may extend the Problem Details document with
```

Comment:
Here "document" becomes deeply confusing.
(Are you talking about the specification or the data item...)

Reply:
Agree, and since you addressed this in [2eac975](https://github.com/core-wg/core-problem-details/pull/16/commits/2eac97594186e4cc158e840d093412b6dedcb84f) we can resolve this comment.